### DISCUSSION TITLE
Support for RISC-V Architecture Issue in Maturin Installation

### DISCUSSION THREAD

**Discussion by gkdlight** [PyO3/maturin#1719] [Q&A]
I'm trying to install maturin using pip, and I have Rust and Cargo already installed. However, during installation, I'm encountering an error that indicates the Rust compiler is not found. The error message is: Failed building wheel for maturin Failed to build maturin, which is required to install pyproject.toml-based projects"

Additionally, I'm like to know if maturin supports the RISC-V architecture. Could you provide guidance on resolving the installation error and share information about RISC-V support in maturin?
![image](https://github.com/PyO3/maturin/assets/61817247/4521ea8a-5b9c-441d-a4e3-9ec2772d5702)

**Comment by messense** [comment]
RISC-V is supported, at least Alpine Linux is distributing maturin for riscv64: https://pkgs.alpinelinux.org/package/edge/community/riscv64/maturin.

I can't reproduce your issue on x86_64 Ubuntu, such issues are highly dependent on system setup so I can't really help you much, you need to investigate by yourself or come up with a clean repro, for example in a Docker container.

**Reply by gkdlight** [reply]
Thank you.
Regarding my environment,I'm not running on Ubuntu x86_64; I'm using Debian on a RISC-V architecture (riscv64 GNU/Linux). Does maturin support this system configuration?

**Comment by konstin** [comment]
Could you run `cargo -vV`? maybe cargo isn't in `PATH` even though rustc is

**Reply by gkdlight** [reply]
![image](https://github.com/PyO3/maturin/assets/61817247/03dcba4d-5896-42a0-9324-8812881c534d)
Result of `cargo -vV`.!
